database/drop_db.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
```

Comment:
An "Are you sure Y/N :" might be worthwhile here

Reply:
done